src/lib/data/near.js

```diff
@@ -337,22 +337,44 @@ async function _initNear({
 }
 
 export const useInitNear = singletonHook({}, () => {
-  const [nearPromise, setNearPromise] = useState(null);
-
-  return {
-    nearPromise,
-    initNear: useMemo(() => (args) => setNearPromise(_initNear(args)), []),
-  };
+    const [nearPromise, setNearPromise] = useState(null);
+
+    return {
+        nearPromise,
+        initNear: useMemo(
+            () => (args) => {
+                const defaultNetworkId = args.networkId;
+                const argConfigs = args.config || {};
+                const testnetArgs = { ...args, config: { ...argConfigs, networkId: 'testnet' } };
+                const mainnetArgs = { ...args, config: { ...argConfigs, networkId: 'mainnet' } };
+                return setNearPromise(Promise.all([testnetArgs, mainnetArgs].map(_initNear)).then((nears) => nears.map((n) => ({ ...n, default: n.config.networkId === defaultNetworkId }))));
```

Comment:
It's better to not copy all args from the config, because config may pass `apiUrl`. Also wallet selector doesn't make sense for a different `networkId`.

Instead initialize selected chain (default) with the given args. And other chain without config specific args.

Reply:
Done ✅